.ci/Jenkinsfile

```
@@ -43,7 +44,14 @@ pipeline {
   parameters {
     booleanParam(name: 'DRY_RUN_MODE', defaultValue: false, description: 'If true, allows to execute this pipeline in dry run mode, without sending a PR.')
     booleanParam(name: 'FORCE_SEND_PR', defaultValue: false, description: 'If true, will force sending a PR, although it could be affected by the value off the DRY_RUN parameter: if the latter is true, a message will be printed in the console.')
-    choice(name: 'APM_AGENTS', choices: ['All','.NET', 'Go', 'Java', 'Node.js', 'PHP', 'Python', 'Ruby', 'RUM'], description: 'Name of the APM Agent you want to update its specs.')
+    choice(name: 'ECS_REPO', choices: ['all',
+                                        'ecs-logging-dotnet',
+                                        'ecs-logging-go-logrus',
+                                        'ecs-logging-java',
+                                        'ecs-logging-js',
+                                        'ecs-logging-python',
+                                        'ecs-logging-ruby',
+                                        'ecs-logging-zap'], description: 'Name of the ECS repo you want to update its specs.')
```

```suggestion
                                        'ecs-logging-go-zap'], description: 'Name of the ECS repo you want to update its specs.')
```

And perhaps move this up to be beside "ecs-logging-go-logrus".